packages/terra-core-docs/src/terra-dev-site/doc/alert/About.1.doc.mdx

```diff
@@ -125,10 +103,9 @@ For further guidance, please view the full [Accessibility Guide](./accessibility
 <InfoExample title="Information Notification Banner" />
 <SuccessExample title="Success Notification Banner" />
 <CustomExample title="Custom Notification Banner" />
-<LongTextExample title="Long Text Notification Banner" />
 <ActionExample title="Notification Banner with Action" />
 <DismissibleExample title="Dismissible Notification Banner" />
-<ActionAndDismissibleExample title="Dismissible Notification Banner with Actions" />
```

Comment:
Removed this since it was a bad example of a custom notification without title and there are already examples of notification with action and dismiss